container/controller/controller/runtime.py

```diff
@@ -107,11 +125,14 @@ class KubeDomainRuntime():
   selector:
     app: compute
     domain: {DOMNAME}
+  type: NodePort
   ports:
   - name: libvirt
     port: 16509
+    nodePort: 30001
```

Comment:
yes we can ,
But thats a next step . 
Since I need some more parsing on cubectl outputs for this.
I still need to find how all rc service and ports describe commands corelate